packages/app/utils/strings.ts

```diff
@@ -0,0 +1,2 @@
+export const capitalize = (word: string) =>
+  `${word.charAt(0).toUpperCase()}${word.slice(1)}`;
```

Comment:
We can do this with css, tailwind has a `capitalize` className.

Reply:
Lol nice, I didn't knew there was a way to do it with CSS, thanks!

Reply:
I think we can delete this file